### PR TITLE
hackerlime- Plugin to push codes directly to hackerrank contests

### PR DESCRIPTION
**Pull request by kaushik94**

### CODE REVIEW

**Comment by FichteFoll** [comment]
- The linux label makes no sense to me.
- `{ "keys": ["ctrl+shift+r"], "command": "goto_symbol_in_project" },` is a default key binding
- Your plugin won't work with ST3, so you need to change the `sublime_text` version selector.
- You'll need to change `"branch": "master"` to `"tags": true` and use tags to create releases. As mentioned [in the docs](https://packagecontrol.io/docs/submitting_a_package#Step_4), we no longer accepts branch-based releases for new packages.

File: repository/h.json [hunk 1]
```diff
@@ -22,6 +22,17 @@
...
+			"name": "Hackerlime",
+			"details": "https://github.com/kaushik94/hackerlime",
+			"labels": ["hackerrank", "linux"],
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "master"
+				}
+			]
+		}
```
Comment (FichteFoll) [inline repository/h.json]: ^ missing comma

**Comment by kaushik94** [comment]
@FichteFoll  thanks for the info. Its my first time, made the changes. Take a look and let me know.

**Comment by FichteFoll** [comment]
You have to push valid [semantic version](http://semver.org) tags (with optional leading `v`) to your repo.

**Comment by kaushik94** [comment]
@FichteFoll  take a look at it now.

**Comment by kaushik94** [comment]
Thanks @FichteFoll

**Pull request merged**